samples/nrf_desktop/configuration/nrf52840_pca10059_ZDebugMCUBoot_bootloader.conf

```diff
@@ -0,0 +1,48 @@
+CONFIG_CONSOLE_HANDLER=y
```

Comment:
I don't like the name of this file.
Maybe `nrf52840_pca10059_mcuboot_ZDebugMCUBoot_.conf`?
I think that we may also change the application config files to `nrf52840_pca10059_app_ZDebugMCUBoot_.conf`

Reply:
done